app/views/categories/show.html.erb

```diff
@@ -1,3 +1,48 @@
 <header>
   <%= render "shared/navbar" %>
-</header>
\ No newline at end of file
+</header>
+<section class="container mt-5 w-75">
+  <%= render "shared/categories" %>
+</section>
+</section>
+<section class="container mb-4">
+  <div class="card p-3 border border-1 rounded-3 shadow-sm w-50 mx-auto">
+    <%= form_tag category_path(@category), :method => 'get', class:'d-flex justify-content-md-end mx-auto' do %>
```

Comment:
let's use `method: 'get'` (actually `method: :get` should work) instead of `:method => 'get'` whenever possible. It was deprecated some time ago

Reply:
Done: b31e20a